### DISCUSSION TITLE
"componentName": "Page", 如何去做扩展Page的属性

### DISCUSSION THREAD

**Discussion by qian-chen97** [opentiny/tiny-engine#147] [General]

[a comment by an automated account is not shown]

**Comment by chilingling** [comment]
1. 非 issue ，转换为 discussions 继续讨论
2. 可以详细描述 拓展 Page 的属性是什么意思吗？具体的场景？诉求是什么？

**Reply by qian-chen97** [reply]
场景如下， 页面去绑定数据模型，关联流程节点，想在这个页面根节点去自定义属性（选择模型，生成类型等），但是看下来Page是直接生成body标签，而不是个组件去渲染

**Reply by chilingling** [reply]
当前我们的页面根节点已更改为 id 为 app 的 div ，可以看看满足场景需求吗？

[a comment by an automated account is not shown]
